### PR TITLE
Integrity hash file

### PR DESCRIPTION
**Pull request by gneyal**

### CODE REVIEW

File: doc/md/concepts/integrity_hash_file.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+id: hash_file
```
Comment (a8m) [inline doc/md/concepts/integrity_hash_file.md:2]: It's a sum_file that contains hashes. 
```suggestion
id: sum_file
```

File: doc/md/concepts/integrity_hash_file.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+slug: /concepts/integrity_hash_file
+title: Integrity Hash File
```
Comment (a8m) [inline doc/md/concepts/integrity_hash_file.md:4]:
```suggestion
title: Integrity Sum File
```

File: doc/md/concepts/integrity_hash_file.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Version Migration Files
+Version migration files are files that describe changes to the databases. Usually these files are prefixed with a number - which describe the version of the database change. The framework that runs these files uses these numbers to determine which changes have been applied to the database.
```
Comment (a8m) [inline doc/md/concepts/integrity_hash_file.md:8]:
```suggestion
## Version Migration Files
Version migration files are files that describe changes to the databases. Usually, these files are prefixed with a number that describes the version of the database change. The framework that executes these script files uses these numbers to determine which files have been applied to the database.
```

File: doc/md/concepts/integrity_hash_file.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+In Ent, you’ll find these files in ent/migrations/ folder.
+In Django, you’ll find these files in project_name/app_name/migrations/ folder.
```
Comment (a8m) [inline doc/md/concepts/integrity_hash_file.md:11]: Why Ent and Django are mentioned here? Feels redundant to me.

File: doc/md/concepts/integrity_hash_file.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## Integrity Hash File
+Databases hold one of the most valuable assets of the organization. Unfortunately, databases are also quite fragile. Especially when working in teams where developers are working in parallel on the same business logic. Data loss is real and that’s why good practice is so important when working with DBs.
```
Comment (a8m) [inline doc/md/concepts/integrity_hash_file.md:14]: Maybe it's just my preferences, but I'd go with the "cut to the chase" approach:

> When multiple engineers work on the same project in parallel, there is a chance that their migration scripts will conflict with each other and may cause unexpected behavior. For example: ...
> ...
>
> Atlas provides a built-in approach to tackle this issue: a migration sum file ...
>
> ### Atlas Sum file
> \<more details about sum files\>
> ..

File: doc/md/concepts/integrity_hash_file.md (hunk 4 above)
Comment (gneyal) [reply doc/md/concepts/integrity_hash_file.md:11]: Will remove this in a later commit.

File: doc/md/concepts/integrity_hash_file.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+id: sum_file
+slug: /concepts/integrity_hash_file
+title: Integrity Sum File
+---
+
+## Version Migration Files
+Version migration files are files that describe changes to the databases. Usually, these files are prefixed with a number that describes the version of the database change. The framework that executes these script files uses these numbers to determine which files have been applied to the database.
+
+In Ent, you’ll find these files in ent/migrations/ folder.
```
Comment (gneyal) [inline doc/md/concepts/integrity_hash_file.md:10]:
```suggestion

```

File: doc/md/concepts/integrity_hash_file.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+In Django, you’ll find these files in project_name/app_name/migrations/ folder.
```
Comment (gneyal) [inline doc/md/concepts/integrity_hash_file.md:11]:
```suggestion

```

File: doc/md/concepts/integrity_hash_file.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Integrity Hash File
+Databases hold one of the most valuable assets of the organization. Unfortunately, databases are also quite fragile. Especially when working in teams where developers are working in parallel on the same business logic. Data loss is real and that’s why good practice is so important when working with DBs.
```
Comment (gneyal) [inline doc/md/concepts/integrity_hash_file.md:14]:
```suggestion
When multiple engineers work on the same project in parallel, there is a chance that their migration scripts will conflict with each other and may cause unexpected behavior. For example:
```

File: doc/md/concepts/integrity_hash_file.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+There are 3 issues that can occur with version migration files.
```
Comment (gneyal) [inline doc/md/concepts/integrity_hash_file.md:16]:
```suggestion

```

File: doc/md/concepts/integrity_hash_file.md (hunk 7 above)
Comment (gneyal) [inline doc/md/concepts/integrity_hash_file.md:11]:
```suggestion

```

**Pull request closed without merging**